setup.py

```diff
@@ -20,9 +20,9 @@
 
 requires = [
     'six',
-    'requests>=2.2.1',
+    'requests>=2.24.0',
```

Comment:
We have this and `requirements.txt` and use tox. Tox should respect these dependencies when testing and this is all pip cares about. Why do we maintain both?

Further I see a `PublishCommand` but we should be using `twine` + PyPI Upload Tokens.  Should we switch to that here too?

Reply:
because travisci doesn't uses requirements.txt and not tox :(

Reply:
Also PublishCommand is using twine.

Reply:
> because travisci doesn't uses requirements.txt and not tox :(

I don't understand this